tools/make_release.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# /**
+#  * ---------------------------------------------------------------------
+#  * GLPI - Gestionnaire Libre de Parc Informatique
+#  * Copyright (C) 2015-2018 Teclib' and contributors.
+#  *
+#  * http://glpi-project.org
+#  *
+#  * based on GLPI - Gestionnaire Libre de Parc Informatique
+#  * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+#  *
+#  * ---------------------------------------------------------------------
+#  *
+#  * LICENSE
+#  *
+#  * This file is part of GLPI.
+#  *
+#  * GLPI is free software; you can redistribute it and/or modify
+#  * it under the terms of the GNU General Public License as published by
+#  * the Free Software Foundation; either version 2 of the License, or
+#  * (at your option) any later version.
+#  *
+#  * GLPI is distributed in the hope that it will be useful,
+#  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  * GNU General Public License for more details.
+#  *
+#  * You should have received a copy of the GNU General Public License
+#  * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+#  * ---------------------------------------------------------------------
+# */
+if [ ! "$#" -eq 2 ]
+then
+ echo "Usage $0 fi_git_dir release";
+ exit ;
+fi
+
+read -p "Are translations up to date? [Y/n] " -n 1 -r
+echo    # (optional) move to a new line
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1 # handle exits from shell or function but don't exit interactive shell
+fi
+
+INIT_DIR=$1;
+RELEASE=$2;
+
+# test glpi_cvs_dir
+if [ ! -e $INIT_DIR ]
+then
+ echo "$1 does not exist";
+ exit ;
+fi
+
+INIT_PWD=$PWD;
+
+if [ -e /tmp/fusioninventory ]
+then
+ echo "Delete existing temp directory";
+\rm -rf /tmp/fusioninventory;
+fi
+
+echo "Copy to  /tmp directory";
+git checkout-index -a -f --prefix=/tmp/fusioninventory/
+
+echo "Move to this directory";
+cd /tmp/fusioninventory;
+
+echo "Check version"
+if grep --quiet $RELEASE setup.php; then
+  echo "$RELEASE found in setup.php, OK."
+else
+  echo "$RELEASE has not been found in setup.php. Exiting."
+  exit 1;
+fi
+if grep --quiet $RELEASE fusioninventory.xml; then
+  echo "$RELEASE found in fusioninventory.xml, OK."
+else
+  echo "$RELEASE has not been found in fusioninventory.xml. Exiting."
+  exit 1;
+fi
+
+echo "Check XML WF"
+if ! xmllint --noout fusioninventory.xml; then
+   echo "XML is *NOT* well formed. Exiting."
+   exit 1;
+fi
+
+echo "Retrieve PHP vendor"
+composer install --no-dev --optimize-autoloader --prefer-dist --quiet
```

Comment:
perhaps delete forlder `vendor` if exist before do this?

Reply:
It does not exists, since the directory in /tmp is created from `git checkout-index` ;)